cdn/signedurls/signurl_test.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"reflect"
+	"testing"
+	"time"
+)
+
+func Test_readKeyFile_wrongPath(t *testing.T) {
+	if _, err := readKeyFile("/non/existing/path"); err == nil {
+		t.Fatal("expected error")
+	}
+}
+
+func Test_readKeyFile_badKey(t *testing.T) {
```

Comment:
This one is also not testing signing.

Reply:
I’ll remove all tests but the SignURL test.